pkg/bootstrapper/kubelet.go

```diff
@@ -6,6 +6,14 @@ import (
 
 	"golang.org/x/sys/windows/svc"
 	"golang.org/x/sys/windows/svc/mgr"
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+const (
+	// waitTimeKubeletService is the time required for kubelet service to go to running state
+	waitTimeKubeletService = 30 * time.Second
```

Comment:
This is not the wait time but rather the interval at which you will poll the service. `svcPollInterval`. Given it is is in `kubelet.go` no reason to have `kubelet`.